reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equality operator."
```

Comment:
Can we be any more specific? The equality operator ==, as most programmers know it, allows the left and right sides to be reversed. That isn't the case with any of these operators.